src/KeyworkerProfile/components/KeyworkerSearch.js

```diff
@@ -20,8 +21,12 @@ class KeyworkerSearch extends Component {
 
 KeyworkerSearch.propTypes = {
   searchText: PropTypes.string,
-  gotoNext: PropTypes.func.isRequired,
+  history: PropTypes.object,
+  handleSearch: PropTypes.func.isRequired,
   handleSearchTextChange: PropTypes.func.isRequired
 };
 
-export default KeyworkerSearch;
+const KeyworkerSearchWithRouter = withRouter(KeyworkerSearch);
```

Comment:
withRouter wrapper gives access to Route props without having to explicitly pass them through the component hierarchy